src/interface/storage.thrift

```diff
@@ -203,6 +205,40 @@ struct CatchUpDataReq {
     3: common.HostAddr     target,
 }
 
+struct UpdateResponse {
+    1: required ResponseCommon result,
+    2: optional common.Schema schema,   // return column related props schema
+    3: optional binary data,            // return column related props value
+    4: optional bool upsert = false,    // it's true when need to be inserted by UPSERT
+}
+
+struct UpdateItem {
+    1: required binary name,    // TagName or EdgeType
```

Comment:
I am not sure how to represent "update vertex id  tag1.prop1 = tag2.prop2 + tag3.prop3" with your UpdateItem

Reply:
name = tag1,  prop = prop1, value = Expression::encode(tag2.prop2 + tag3.prop3)